hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockRecovery1.java

```diff
@@ -122,9 +108,9 @@
 /**
  * This tests if sync all replicas in block recovery works correctly.
  */
-public class TestBlockRecovery {
+public class TestBlockRecovery1 {
```

Comment:
Lets not bother the name, Let it be as is.

Reply:
Sure, sounds good. Will update.